okta/user_schema.go

```diff
@@ -150,6 +150,12 @@ var (
 			ValidateFunc:  validation.StringInSlice([]string{"UNIQUE_VALIDATED", "NOT_UNIQUE"}, false),
 			ConflictsWith: []string{"one_of", "enum", "array_type"},
 		},
+		"user_type": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Custom Subschema usertype",
```

Comment:
@bogdanprodan-okta Minor, make it `user type` and just `subschema` in the Description. 😉